### PR TITLE
add message for completed inspections

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

https://github.com/thoth-station/thoth-application/issues/398

## This introduces a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

File: thoth/messaging/inspection_complete.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+class InspectionCompletedMessage(MessageBase):
+    """Class used to indicate when amun inspections complete."""
+
+    topic_name = "thoth.inspection-completed"
+    _message_version = 1  # update on schema change
+
+    class MessageContents(BaseMessageContents, serializer="json"):  # type: ignore
+        """Class used to represent contents of a inspection-completed message Kafka topic."""
+
+        inspection_id: str
```
Comment (fridex) [inline thoth/messaging/inspection_complete.py:33]: I wonder if we should add also Amun API URL and deployment name from the configmap, so we can track down the cluster:

https://github.com/thoth-station/thoth-application/blob/4ca617b49cead4cd6650cdda39fc562b94e7c3ae/amun/overlays/amun-api/configmaps.yaml#L7-L8

Comment (goern) [reply thoth/messaging/inspection_complete.py:33]: Isn't that part of an inspection?

Comment (fridex) [reply thoth/messaging/inspection_complete.py:33]: It is. If we want to obtain the inspection result based on `inspection_id` on Thoth side, we need to know which Amun API/deployment should be reached.

Comment (goern) [reply thoth/messaging/inspection_complete.py:33]: chk

**Review by fridex — APPROVED** [review]
:+1:

**Pull request merged**